sdk/src/sysvar/recent_blockhashes.rs

```diff
@@ -162,20 +215,24 @@ mod tests {
                 })
             })
             .collect();
-        unsorted_recent_blockhashes.shuffle(&mut thread_rng());
+        blocks.shuffle(&mut thread_rng());
```

Comment:
nits: rename to `unsorted_blocks` so that this like looks sensible, meaning basically carrying the `unsorted_` from `unsorted_recent_blockhashes` to `blocks`.